lib/client/Client.js

```diff
@@ -536,6 +529,31 @@ export default class Client {
     return this.updateMessage(id, { mailboxIds: mailboxIds });
   }
 
+  _createMessage(message, role, isDraft) {
```

Comment:
Code factoring.I'm ding the `isDraft` assignation only here and the calling methods are onliners.